packages/sdk/README.md

```diff
@@ -87,7 +89,8 @@ When both, `uri` and `options` (or `service`) are present, the `uri` takes prece
 {
   service: String,
   options: Object,
-  payload: Object | String
+  uriPrefix?: String,
```

Comment:
Given we want to remove it should we not document it?

Reply:
🤔

What we want to remove is the legacy MC API endpoint...

Reply:
Ah, got yah. I thought you, given we migrated all routes, also want to remove the option.